plugin/hpsa/hpsa.py

```diff
@@ -134,6 +151,7 @@ def _parse_hpssacli_output(output):
             else:
                 cur_data_pointer[cur_line_splitted[0].lstrip()] = \
                     ": ".join(cur_line_splitted[1:]).strip()
+
```

Comment:
Ok, I guess I misinterpreted the diff (it looked like the diff showed that the newline existed already, implying that there was new whitespace "junk" in the line). If it's a clean newline, no concerns on my end.